content/blog/tidymodels-survival-analysis/index.Rmd

```diff
@@ -0,0 +1,129 @@
+
+We could look at the time and model that as a regression problem. We could look at the event status and model that as a classification problem. Both options might get us somewhere close to an answer to our original modeling question but not quite there. Survival analysis with its censored regression models lets us model an outcome that includes both aspects, the time and the event status. And with that, it can deal with both censored and uncensored observations appropriately. With this type of model, we can predict the survival time, or in more applied terms, how long someone will stay a customer for overall. We can also predict the probability of survival at a given time point. This let's us answer questions like "How likely is it that this customer will churn after 3 months?". See which prediction types are available for which models at [censored.tidymodels.org](https://censored.tidymodels.org/).
+
+
+## Ch-ch-changes: What's new for censored regression?
+
+The main components needed for this full-fledged integration of survival analysis into tidymodels were
+
+- Survival analysis models that can take censoring into account
+- Survival analysis performance metrics that can take censoring into account
+- Integrating changes required by these models and metrics into the framework
+
+For the models, parsnip gained a new mode, `"censored regression"`, for existing models as well as new model types such as `proportional_hazards()`. Engines for these reside in the parsnip extension package censored. This has been around for a while and we've previously shared [our inital thoughts](https://www.tidyverse.org/blog/2021/11/survival-analysis-parsnip-adjacent/) and the [release of censored](https://www.tidyverse.org/blog/2022/08/censored-0-1-0/).
+
+Now we've added the metrics: yardstick v1.3.0 includes new metrics for assessing censored regression models. Somewhat similar to how metrics for classification models can take class predictions or probability predictions as input, these survival metrics can take predicted survival times or predictions of survival probabilities as input. 
+
+The new metrics are
+
+- Concordance index on the survival time via `concordance_survival()`
+- Brier score on the survival probability and its integrated version via `brier_survival()` and `brier_survival_integrated()`
+- Area under the ROC curve on the survival probabilities via `auc_roc_survival()`
+
+The probability of survival is always defined _at a certain point in time_. We call that time point the evaluation time because it is then also the time point at which we want to evaluate model performance. Metrics that work on the survival probabilities are also called _dynamic metrics_ and you read more about them here:
```

Comment:
```suggestion
The probability of survival is always defined _at a certain point in time_. We call that time point the evaluation time because it is then also the time point at which we want to evaluate model performance. Metrics that work on the survival probabilities are also called _dynamic metrics_ and you can read more about them here:
```